17/umbraco-cms/fundamentals/setup/server-setup/load-balancing/load-balancing-backoffice.md

```diff
@@ -0,0 +1,49 @@
+# Load Balancing the Backoffice
+
+This article contains specific information about load balancing the Umbraco backoffice, ensure you read the [Load Balancing Overview](./) and relevant articles about general load balancing principles before you begin.
+
+By default, the Umbraco load balancing setup assumes there are a single backoffice server, and multiple front-end servers. From version 17 it's possible to load balance the backoffice, meaning there's no need to differentiate from backoffice servers and front-end servers, however, this requires some additional configuration steps.
+
+## Server Role Accessor
+
+Umbraco has the concept of server roles, to differentiate between backoffice servers and front-end servers. Since all servers will be backoffice servers, we need to add a custom `IServerRoleAccessor` to specify this.
+
+Start by implementing a custom `IServerRoleAccessor` that pins the role as `SchedulingPublisher`:
+
+```csharp
+ public class StaticServerAccessor : IServerRoleAccessor
+{
+    public ServerRole CurrentServerRole => ServerRole.SchedulingPublisher;
+}
+```
+
+You can now register this accessor, either in `Program.cs` or via a Composer:
+
+```csharp
+umbracoBuilder.SetServerRegistrar(new StaticServerAccessor());
+```
+
+This will ensure that all servers are treated as backoffice servers.
+
+## Load balancing Isolated Caches
```

Comment:
Should we use "repository caches" over "isolated caches" - in the heading and text?  Just for me that seems like a more easily understood term - but if you disagree it's fine as is.